platform/ios/Integration Tests/MGLEmptyStopsTests.m

```diff
@@ -0,0 +1,61 @@
+#import "MGLMapViewIntegrationTest.h"
```

Comment:
There is no need to add an additional test class. All `Expressions` tests are covered in `MGLExpressionTests.mm` and `MGLPredicateTests.mm` respectively.

Reply:
Agreed, this seems like a pretty roundabout way of testing what’s ultimately a minor, pretty self-contained edge case. But if generalized a bit, it could serve as a much more useful integration test of runtime styling.

Reply:
I'm in two minds about this. The test was written first as an integration test to try to capture the specific crash (during rendering) mentioned in the original ticket. Recent changes appear to no longer trigger the crash, so this specific test ends up being a somewhat bloated unit-test (and essentially a duplicate of the tests in `MGLExpressionTests.mm`.

I'm tempted to leave it - in case of regression (though perhaps tweaked, and the filename changed) so that it becomes a home for future styling integration tests (as @1ec5 commented).